web/viewer.html

```diff
@@ -176,6 +176,12 @@
             <div id="editorHighlightColorPicker" class="colorPicker">
               <span id="highlightColorPickerLabel" class="editorParamsLabel" data-l10n-id="pdfjs-editor-highlight-colorpicker-label">Highlight color</span>
             </div>
+            <div id="editorHighlightThickness">
+              <label for="editorFreeHighlightThicknessInput" class="editorParamsLabel" data-l10n-id="pdfjs-editor-free-highlight-thickness-input">Thickness</label>
+              <div class="thicknessPicker">
+                <input type="range" id="editorFreeHighlightThicknessInput" class="editorParamsSlider" value="12" min="8" max="24" step="1" tabindex="101">
```

Comment:
Here, and throughout the JS code, can we use a slightly shorter id (since no other editor seem to include "Input" in its name)?
```suggestion
                <input type="range" id="editorFreeHighlightThickness" class="editorParamsSlider" value="12" min="8" max="24" step="1" tabindex="101">
```